api/XRSystem.json

```diff
@@ -5,10 +5,15 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRSystem",
         "spec_url": "https://immersive-web.github.io/webxr/#xrsystem-interface",
         "support": {
-          "chrome": {
-            "alternative_name": "XR",
-            "version_added": "79"
-          },
+          "chrome": [
+            {
+              "version_added": "83"
+            },
+            {
+              "alternative_name": "XR",
+              "version_added": "79"
```

Comment:
The `XR` name was not kept around as an alias, so `version_removed` needed here.

Reply:
I think it is kept around still https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/xr/xr_system.idl;l=18?q=xrsystem%20lang:idl&ss=chromium

Reply:
That's a typedef, and that doesn't cause a `window.XR` interface object to exist, it's just an alias so that all instances of `XR` in Chromium's IDL didn't need to be renamed at once.

You can test this by evaluating `window.XR` and `window.XRSystem`.

Reply:
ahh okay, thanks!